ee/localserver/request-query.go

```diff
@@ -0,0 +1,49 @@
+package localserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func (ls *localServer) requestQueryHandler() http.Handler {
+	return http.HandlerFunc(ls.requestQueryHanlderFunc)
+}
+
+func (ls *localServer) requestQueryHanlderFunc(res http.ResponseWriter, r *http.Request) {
+	if r.Body == nil {
+		res.Write([]byte("request body is nil"))
+		return
+	}
+
+	var body map[string]string
+	if err := json.NewDecoder(r.Body).Decode(&body); err != nil {
+		res.Write([]byte(fmt.Sprintf("error unmarshaling request body: %s", err)))
+		return
+	}
+
+	query, ok := body["query"]
+	if !ok {
+		res.Write([]byte("no query key found in request body json"))
+		return
+	}
+
+	if query == "" {
+		res.Write([]byte("empty query"))
+		return
+	}
+
+	results, err := ls.querier.Query(query)
```

Comment:
Do we need a timeout here? I forget if there's one implemented at a lower level. (I know the http server will time out, but will the query?)

Reply:
added timeout ... unsure of reasonable duration interval, went with 1 sec && 250 milliseconds